pagina-practica/practica.html

```diff
@@ -21,7 +21,55 @@
             <a id="enlaces" href="#contactos">Contactos</a>
         </div>
     </div>
-    
-    
+
+    <section class="home" id="home">
+        <img id="img" src="../pagina-practica/imagenes/undraw_Experience_design_re_dmqq.png" alt="Home">
+    </section>
+
+    <article class="about" id="abouts">
+        <div>
+            <img id="imagen-about" src="../pagina-practica/imagenes/undraw_online_stats_0g94.png" alt="About">
+        </div>
+
+        <div>
+            <h3 id="title">Computing</h3>
+            <p id="text">Computer Science is the branch<br>
+             of Engineering that studies<br>
+             the hardware, data networks and<br>
+             software necessary to process information<br>
+             automatically.</p><br>
+        </div>
+
+    </article>
+
+    <footer class="contacto" id="contactos">
+        <div id="contacto">
+            <h3>Contacto</h3>
+            <img id="imagen-about"src="../pagina-practica/imagenes/undraw_mobile_analytics_72sr.png" alt="">
+        </div>
+
+        <div class="formulario">
+            
+                <div class="informacion">
+                    <label for="name">Name:</label>
+                    <input type="Nombre" id="name">
+                
+                    <label for="last name">Last-name:</label>
+                    <input type="Last name" id="last name">
+                
+                    <label for="menssage">Comment:</label>
+                    <textarea id="menssage" name="user_message"></textarea>
+               
+                <div class="button">
+                    <button type="siguiente">Start</button>
+
+                </div>
```

Comment:
Always ident the code, there is not very clear which div element are you closing also it is a good practice.